aidial_adapter_vertexai/chat/gemini/adapter.py

```diff
@@ -321,6 +330,27 @@ async def create_function_calls(
             )
 
 
+async def create_grounding(candidate: Candidate, consumer: Consumer) -> None:
+    # TODO:  candidate.grounding_metadata.grounding_supports[i].segment
```

Comment:
You could incorporate the search results in a dedicated stage called `Google Search`

Reply:
Created issue
https://github.com/epam/ai-dial-adapter-vertexai/issues/150

Reply:
Why do we need to repeat verbatim in the source code the content of the issue we have in GitHub? 
It's either one of the two _(TODO comment or an issue)_, no need to have both. 

You could reference the relevant piece of code in the issue itself.